dist/src/main/resources/rules/file-mappings.windup.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="XmlFileMappings">
+    <rules>
+        <rule>
```

Comment:
I don't think that I understand. file-mapping is a Rule, so putting it inside of a Rule doesn't completely make sense? I'm actually a little surprised that it works. AFAIC, the only reason that it does is that the RuleHandler class is a little lax in its handling of subelements.